Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM node:5-slim
+ENV VERSION=1.3.0
+WORKDIR /srv/app
+ADD . /srv/app
+RUN npm pack && npm install -g kongfig
```

Comment:
`npm install --global --production kongfig`...and you need a new line at the end of the file :wink: